code/game/objects/structures/transit_tubes/transit_tube_construction.dm

```diff
@@ -0,0 +1,110 @@
+/obj/structure/transit_tube_construction
+	name = "uninstalled transit tube segment"
+	desc = "An uninstalled piece of a transit tube network."
+	anchored = FALSE
+	density = FALSE
+
+	icon = 'icons/obj/pipes/transit_tube_rpd.dmi'
+
+	var/installed_type = null
+	var/installed_type_flipped = null
+	var/flipped = FALSE
+
+/obj/structure/transit_tube_construction/proc/rotate()
+	setDir(turn(dir, -90))
+
+/obj/structure/transit_tube_construction/update_icon_state()
+	icon_state = "[base_icon_state][flipped ? "_flipped" : ""]"
+
+/obj/structure/transit_tube_construction/proc/flip()
+	if(!installed_type_flipped)
+		return
+
+	flipped = !flipped
+	update_icon_state()
+
+/obj/structure/transit_tube_construction/screwdriver_act(mob/living/user, obj/item/I)
+	. = TRUE
+	var/turf/T = get_turf(src)
+	if(!isfloorturf(T))
+		to_chat(user, "<span class='notice'>You can only install [src] on a floor.</span>")
+		return
+	for(var/obj/turf_contents in T)
+		var/is_tube_part = (istype(turf_contents, /obj/structure/transit_tube_construction) || istype(turf_contents, /obj/structure/transit_tube))
+		if(!(is_tube_part) && turf_contents.density >= 1)
```

Comment:
Please ensure this function probably works. Looking at it suggests it will succeed if there's a transit tube there. 
Density is also true/false, Remove the >= 1 and just replace it with a val check

Reply:
Yeah, this logic was trying to do two things at once: prevent tube pieces from being installed over each other/walls, but allow pods to be installed on pre-existing tubes. I've separated these out into two pieces of implementation.

Reply:
Done, PTAL